models/dynamics_365_crm.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+
+models:
+  - name: account
```

Comment:
We should consider adding some data integrity tests here? Maybe we can leverage [equal_rowcount](https://github.com/dbt-labs/dbt-utils?tab=readme-ov-file#equal_rowcount-source) to ensure there's no fanout following the (un)pivots from the macro.